railties/lib/rails/generators/test_case.rb

```diff
@@ -25,7 +25,7 @@ module Generators
     #     destination File.expand_path("../tmp", File.dirname(__FILE__))
     #     setup :prepare_destination
     #   end
-    class TestCase < ActiveSupport::TestCase
+    class TestCase < ApplicationTestCase
```

Comment:
It still means that Rails must provide the structure for that in generators, yes?